application/src/main/resources/thingsboard.yml

```diff
@@ -710,6 +710,8 @@ transport:
     dtls:
       # Enable/disable DTLS 1.2 support
       enabled: "${COAP_DTLS_ENABLED:false}"
+     # RFC7925_RETRANSMISSION_TIMEOUT_IN_MILLISECONDS = 9000
+      retransmission_timeout: "${COAP_DTLS_RETRANSMISSION_TIMEOUT_IN_MILLISECONDS:9000}"
```

Comment:
Please take a look how we name parameters that ar configured in milliseconds. For example:
```
stats_print_interval_ms: "${SQL_ATTRIBUTES_BATCH_STATS_PRINT_MS:10000}"
```

There are 77 parameters which env variable names end with "_MS" and your "_MILLISECONDS" is unique. Rename the parameter.